block_qr.php

```diff
@@ -0,0 +1,191 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Class block_qr
+ *
+ * @package    block_qr
+ * @copyright  2023 ISB Bayern
+ * @author     Florian Dagner <florian.dagner@outlook.de>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class block_qr extends block_base {
+    function init() {
+        $this->title = get_string('pluginname', 'block_qr');
+    }
+
+    function get_content() {
+        global $OUTPUT, $CFG, $PAGE, $course;
```

Comment:
```suggestion
        global $OUTPUT, $CFG;
```
$PAGE and $course are unused